buildpack/files.go

```diff
@@ -3,8 +3,13 @@
 package buildpack
 
 import (
+	"errors"
 	"fmt"
+	"os"
 
+	"github.com/buildpacks/lifecycle/buildpack/layertypes"
+	v05 "github.com/buildpacks/lifecycle/buildpack/v05"
+	v06 "github.com/buildpacks/lifecycle/buildpack/v06"
```

Comment:
```suggestion
	"github.com/buildpacks/lifecycle/buildpack/v05"
	"github.com/buildpacks/lifecycle/buildpack/v06"
```

Reply:
Also maybe `api05` and `api06` would be better names?

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/537/commits/53c486316a8f616bd11b4c5ecb4a2a304c6b9a93.